components/SwipeRow.js

```diff
@@ -326,6 +384,17 @@ class SwipeRow extends Component {
                         ? 0
                         : this.props.leftOpenValue;
                 }
+                if (
+                    this.currentTranslateX - projectedExtraPixels >
+                    this.props.leftActivationValue
+                ) {
+                    // we're more than halfway
```

Comment:
I think this comment is misleading - I think it should say something like "We've passed the threshold to trigger the leftActionValue" .. something like that